### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by PeruDayani**
Minor fixes between the frontend and backend:
- [x] Sensor UI spacing and colliders
- [x]  Connect the oculus button to sensor UI cycling
- [x] Fix home waypoint logic. 
- [x] Drone sim cleanup

### CODE REVIEW

**Review by jasmine-bae — COMMENTED** [review]
Everything looks good to me.

File: Assets/Scripts/Drone Scripts/DroneSimulationManager.cs [hunk 1]
```diff
@@ -6,143 +6,340 @@
 public class DroneSimulationManager : MonoBehaviour {
 
     [Header("Selected drone and simulation speed")]
-    public Drone drone;
-    public float speed = 1.0f;
+    private Drone drone;
```
Comment (erwang01) [inline Assets/Scripts/Drone Scripts/DroneSimulationManager.cs:9]: woah this is a massive change to the file.

File: Assets/Scripts/ROS Scripts/ROS Connections v2/ROSManager.cs [hunk 2]
```diff
@@ -109,11 +109,6 @@ private void InstantiateDrone(ROSDroneConnectionInput rosDroneConnectionInput)
         GameObject droneGameObject = droneInstance.gameObjectPointer;
         droneGameObject.name = rosDroneConnectionInput.droneName;
 
-        // Add drone sim manager script on the drone
-        DroneSimulationManager droneSim = droneGameObject.AddComponent<DroneSimulationManager>();
```
Comment (erwang01) [inline Assets/Scripts/ROS Scripts/ROS Connections v2/ROSManager.cs:113]: is the simulation Manger added somewhere else now?

**Review by erwang01 — APPROVED** [review]
Looks good to me. didn't test though. Also I had a question about where the sim manager was generated.

File: Assets/Scripts/ROS Scripts/ROS Connections v2/ROSManager.cs (hunk 2 above)
Comment (PeruDayani) [reply Assets/Scripts/ROS Scripts/ROS Connections v2/ROSManager.cs:113]: yes it's on the drone prefab

File: Assets/Scripts/Drone Scripts/DroneSimulationManager.cs (hunk 1 above)
Comment (PeruDayani) [reply Assets/Scripts/Drone Scripts/DroneSimulationManager.cs:9]: ahahah yeah

**Pull request merged**